timescaledb/getting-started/query-data.md

```diff
@@ -1,153 +1,151 @@
+   
+   For example, say you have a table with timestamp column `time`, and numeric column `value`. If you 
+   used `first(value, time)`, your results would show the first number in the `value` column when ordered 
+   with respect to the `time` column ascending in value. 
+
+   In this query, you will use both the `first()` and `last()` functions in order to find the 
+   first and last trading price for each company. With the `where` clause, the interval of time is 
+   limited to the last three days. 
+
+   ```sql
+   select symbol, first(price,time), last(price, time)
+   from stocks_real_time srt
+   where time > now() - interval '3 days'
+   group by symbol;
+   ```
+
+1. Using the `time_bucket()` function
+
+   Before showing the query, let's look at the `time_bucket()` function in a more detail. 
+
+   The `time_bucket()` function allows you to take a time column and “bucket” the values 
+   based on an interval of your choice. Often you "bucket" time so that you can perform
+   an aggregation over the chosen interval. 
+   
+   For example, say `time` is a timestamp column that shows values incrementing every hour 
+   and `value` is a numeric column. You would like to aggregate `value` using the `sum()` 
+   function so that you get summed values over each day. To accomplish this you can
+   use the `time_bucket()` function on the `time` column to “bucket” the hourly data into daily data 
+   then perform the `sum()` function on the `value` column to get the sum of your values across each day. 
+
+   <img class="main-content__illustration"
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/time-bucket.jpg"
+    alt="time_bucket() Illustration"/>
+
+   For more information on the `time_bucket()` function, see the [`time_bucket()` page][time-bucket] within our API 
+   Reference section. 
+
+   For this SQL query, you will find the average stock trading price for each company
+   over each day. We use the `time_bucket()` function with an interval of one day, include 
+   the `avg()` function on price, and include `symbol` so we can group the average 
+   price calculations by company. The `where` clause specifies results will be limited to only 
+   show days that occur within the last week. The [`group by`][clause-expressions] clause is necessary 
+   for aggregation and allows you to group the results by the day and company. And finally the `order by` 
+   clause orders the results first on the bucketed date, then by symbol. 
+
+   ```sql
+   select
+      time_bucket('1 day', time) as bucket,
+      symbol,
+      avg(price)
+   from stocks_real_time srt
+   where time > now() - interval '1 week'
+   group by bucket, symbol
+   order by bucket, symbol;
+   ```
+
+</procedure>
+
+<highlight type="note">
+For more information about the functions provided by TimescaleDB and Timescale Toolkit extension,
+see the [API Reference for hyperfunctions](/api/:currentVersion:/hyperfunctions).
+</highlight>
```

Comment:
```suggestion
For more information about the functions provided by TimescaleDB and Timescale Toolkit extension,
see the [API Reference for hyperfunctions](/api/:currentVersion:/hyperfunctions).
```

Reply:
Edited Lana's suggestion to say "TimescaleDB Toolkit." I'm not actually sure how we brand it, but that is what we have on the main Toolkit docs page right now.

Also edited link because `:currentVersion:` doesn't work in highlights (highlights are parsed in Markdown separately from the main page, as are tags, which creates all sorts of funny limitations)